src/main/java/org/folio/circulation/domain/policy/LoanPolicy.java

```diff
@@ -118,19 +118,11 @@ public static LoanPolicy unknown(String id) {
       final Result<DateTime> proposedDueDateResult =
         determineStrategy(true, systemDate).calculateDueDate(loan);
 
-      final JsonObject loansPolicy = getLoansPolicy();
-
-      if (proposedDueDateResult.failed() && isFixed(loansPolicy)) {
+      if (proposedDueDateResult.failed()) {
```

Comment:
Does this mean that the decision to allow overrides is not dependent upon the type of policy?

Reply:
In my understanding, on the basis of comments in CIRC-319 and correspondence with Emma, ​​yes.

Reply:
Thanks